arch/arm/boot/dts/bcm283x-rpi-lan7515.dtsi

```diff
@@ -36,9 +46,11 @@
 
 / {
 	__overrides__ {
-		eee = <&ethernet>,"microchip,eee-enabled?";
-		tx_lpi_timer = <&ethernet>,"microchip,tx-lpi-timer:0";
-		eth_led0 = <&ethernet>,"microchip,led-modes:0";
-		eth_led1 = <&ethernet>,"microchip,led-modes:4";
+		eee = <&eth_phy>,"microchip,eee-enabled?";
+		tx_lpi_timer = <&eth_phy>,"microchip,tx-lpi-timer:0";
+		eth_led0 = <&eth_phy>,"microchip,led-modes:0";
+		eth_led1 = <&eth_phy>,"microchip,led-modes:4";
+		eth_downshift_after = <&eth_phy>,"microchip,downshift-after";
+		eth_max_speed = <&eth_phy>,"max-speed";
```

Comment:
Have I missed this eth_max_speed parameter somewhere? I don't see it in the binding nor bcm283x-rpi-lan7515.dtsi

Reply:
The eth_max_speed parameter is defined above, but the `max-speed` property it sets is a common to all(?) PHYs, hence the lack of a vendor prefix.

Reply:
Probably my misunderstanding then.
I thought the overrides section could only change existing parameters. I can't see max-speed having been declared in the eth-phy node, so was confused as to what it was going to be changing. Can overrides add nodes?

Reply:
Overrides can add properties, and by enabling fragments they can add nodes.